### ISSUE TITLE
Minikube start throwing error Failed to enable unit: Unit file cri-docker.socket does not exist.

### ISSUE THREAD

**Issue by Abhijitrb** [kubernetes/minikube#14486]
### What Happened?

loadtest@COMP-4928-1 ~]$ minikube start
\U0001f604  minikube v1.26.0 on Redhat 8.2
\u2728  Using the none driver based on existing profile
\U0001f44d  Starting control plane node minikube in cluster minikube
\U0001f504  Restarting existing none bare metal machine for "minikube" ...
\u2139\ufe0f  OS release is Red Hat Enterprise Linux 8.2 (Ootpa)

\u274c  Exiting due to RUNTIME_ENABLE: sudo systemctl enable cri-docker.socket: exit status 1
stdout:

stderr:
Failed to enable unit: Unit file cri-docker.socket does not exist.

### Attach the log file

![Cri-Docker error](https://user-images.githubusercontent.com/108261750/176832490-78161fa8-9e08-4519-93fe-aa05853b7ca0.JPG)

### Operating System

Redhat/Fedora

**Comment by spowelljr** [comment]
Hi @Abhijitrb, thanks for reporting your issue with minikube.

This is a duplicate of https://github.com/kubernetes/minikube/issues/14410

You can read this comment (https://github.com/kubernetes/minikube/issues/14410#issuecomment-1171741503) for more info. But the quick answer is:

> Kubernetes v1.24 dropped support for Dockershim, so if you want to use the combination of the none driver, Kubernetes v1.24+, and the Docker container runtime you'll need to install cri-dockerd on your system, as listed in our [requirements page](https://minikube.sigs.k8s.io/docs/drivers/none/#requirements).

So follow these install instructions and your issue will be resolved: https://github.com/Mirantis/cri-dockerd#build-and-install

I'm going to close this issue to prevent duplicates, but if you have follow up questions please ask them in https://github.com/kubernetes/minikube/issues/14410, thanks

**Comment by Abhijitrb** [comment]
Thank you Steven for your comments!

I followed that solution link #14410. In that I tried to install cri-dockerd
After executing these commands 
# Run these commands as root
###Install GO###
wget https://storage.googleapis.com/golang/getgo/installer_linux
chmod +x ./installer_linux
./installer_linux
source ~/.bash_profile

I don't see any directory created with name ****cri-dockerd****. Due to this I can't execute cd cri-dockerd to execute remaining commands. 
Not sure whats going wrong here!
After executing source ~/.bash_profile where will it create directory named "cri-dockerd"?

Please let me know

**Comment by Abhijitrb** [comment]
Yes I am running these command as root. I tried with  "sudo systemctl enable cri-docker.service" still i can see same error
Failed to enable unit: Unit file cri-docker.service does not exist

Till  "systemctl daemon-reload" this point there were no issues

Not able to understand those services are present still why enable not works!

**Comment by Abhijitrb** [comment]
This issue got resolved now..After that when I re-executed  minikube start --driver=none a new issue has been encountered
 Exiting due to RUNTIME_ENABLE: Temporary Error: sudo crictl version: exit status 1
stdout:

stderr:
sudo: crictl: command not found


After this I again tried to install  crictl  package however not worked!

Same issue coming

**Comment by spowelljr** [comment]
What happens when you try to run `sudo crictl`?

**Comment by Abhijitrb** [comment]
On running "sudo crictl" not getting any response
sudo: crictl: command not found

However I searched for crictl package and found link for crictl installation.
 https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md

I installed but no success so far. Not able to figure out why this error coming!


here is the complete error message::

**Exiting due to RUNTIME_ENABLE: Temporary Error: sudo crictl version: exit status 1
stdout:

stderr:
sudo: crictl: command not found**
